components/camel-google/camel-google-pubsub-lite/src/main/java/org/apache/camel/component/google/pubsublite/GooglePubsubLiteProducer.java

```diff
@@ -59,6 +59,9 @@ public GooglePubsubLiteProducer(GooglePubsubLiteEndpoint endpoint) {
     public void process(Exchange exchange) throws Exception {
 
         if (logger.isDebugEnabled()) {
+            // TODO Update once basline is Java 21
```

Comment:
typo basline --> baseline , I guess it wil lbe useful to have the correct workding to easily search for similar todos